doc/conf.py

```diff
@@ -22,9 +22,9 @@
 
 # -- Project information -----------------------------------------------------
 
-project = u'genesis-client'
-copyright = u'2019, Mario Juric'
-author = u'Mario Juric'
+project = u'adc-streaming'
+copyright = u'2020, Astronomy Data Commons Team'
+author = u'Astronomy Data Commons Team'
```

Comment:
Let me know if this should be something different.

Reply:
LGTM